README.md

```diff
@@ -26,3 +26,19 @@ Set up secrets on your newly created repository by navigating to the `Settings`
 ## Verify Builds are Successful
 
 At this point, the Workflow should complete successfully. If builds are failing, ensure that your secrets are properly set and that your deployment has finished activating.
+
+## Developing Locally
```

Comment:
Instead of "Developing Locally" I'd call this "Next Steps" and include a preamble like:

Now that your GitHub repository is setup with CI/CD to deploy to Dagster Cloud, you can add your own Dagster code. To run this project locally...

And then at the end maybe something like:

You can also copy any existing [Dagster examples or quickstart projects](https://github.com/dagster-io/dagster/tree/master/examples) into this GitHub repository.